src/ceph_mon.cc

```diff
@@ -272,25 +272,14 @@ int main(int argc, const char **argv)
 
   // We need to specify some default values that may be overridden by the
   // user, that are specific to the monitor.  The options we are overriding
-  // are also used on the OSD (or in any other component that uses leveldb),
-  // so changing the global defaults is not an option.
+  // are also used on the OSD, so changing the global defaults is not an option.
```

Comment:
OK, so the least thing the comment is about is `keyring`, right?

Reply:
sorry, maybe i do not get the point, Do you mean we should make more clear on the comment?